code/modules/clothing/under/syndicate.dm

```diff
@@ -240,7 +240,7 @@
 	icon_state = "inteqeng"
 	supports_variations = KEPORI_VARIATION | VOX_VARIATION | DIGITIGRADE_VARIATION
 
-/obj/item/clothing/under/syndicate/inteq/skirt/artificer
+/obj/item/clothing/under/syndicate/inteq/artificer/skirt
```

Comment:
Base inteq skirt has `body_parts_covered = CHEST|GROIN|ARMS`, so it doesn't cover legs. Should probably copy it over, just so it's still the same.